indico/modules/events/papers/controllers/management.py

```diff
@@ -35,9 +40,11 @@ class RHPapersDashboard(RHManagePapersBase):
 
     def _process(self):
         if not self.event_new.has_feature('papers'):
-            return WPManagePapers.render_template('management/disabled.html', self._conf, event=self.event_new)
+            return WPManagePapers.render_template('management/disabled.html', self._conf, event=self.event_new,
+                                                  cfp=self.event_new)
```

Comment:
Yeah, there is no good reason to pass it at all - just use `event.cfp` in the template or if you want it shorter add a `{% set cfp = event.cfp %}` in the template